crates/cxx-qt-gen/src/naming/cpp.rs

```diff
@@ -242,7 +242,10 @@ fn path_segment_to_string(segment: &PathSegment, type_names: &TypeNames) -> Resu
             ));
         }
     } else {
-        type_names.cxx_qualified(&segment.ident)
+        if let Some(ty) = possible_built_in(&segment.ident.to_string()) {
```

Comment:
which case is this hitting?

Reply:
Done

Reply:
Also refactored the function a bit, I think it's a bit cleaner and more functional now :)